lib/LaTeXML/Package/physics.sty.ltxml

```diff
@@ -202,11 +187,17 @@ DefMacro('\ordersymbol', '\mathcal{O}');
 # 2 TeX (only) required arguments, fences specified
 DefMacro('\lx@physics@fencedII{}{}{}{}{}', sub {
     my ($gullet, $cs, $semantic, $function, $open, $close) = @_;
-    my $size = phys_readDelimiterSize($gullet);
+    my $size = phys_readSize($gullet);
     my $arg1 = $gullet->readArg();
     my $arg2 = $gullet->readArg();
-    return phys_applyFunction($semantic && Invocation(T_CS('\@CSYMBOL'), $semantic),
-      $function, $size, $open, $close, $arg1, $arg2); });
+    return BuildXMDual(
+      { reversion => Tokens($cs, phys_revSize($size),
+          phys_revArg(T_XMArg(1)), phys_revArg(T_XMArg(2))) },
+      Invocation(T_CS('\@APPLY'), Tokens(Invocation(T_CS('\@CSYMBOL'), $semantic),
+          T_XMArg(1), T_XMArg(2))),
+      Tokens(phys_open($size, $open), T_XMArg(1),
+        T_OTHER(','), T_XMArg(2), phys_close($size, $close)),
+      $arg1, $arg2); });
```

Comment:
Looking at it used, I volunteer a renaming from `T_XMArg` to `T_Arg`, much easier on the eyes. I can change to that in my PR as well.

Reply:
I'm confused. Is it `T_XMArg` that you're using, or a shorthand for `#1`,... that you're using?

Reply:
My expl3 experimental code is currently doing the following as a new exported sub of Token.pm:
```perl
sub T_OUT_PARAM {
  my ($v) = @_;
  my $int = $v;
  if (ref $v eq 'LaTeXML::Core::Token') {    # get the integer value from the token
    $int = ord($$v[0]) - ord('0');
    if ($int < 1 || $int > 9) {
      Fatal('malformed', 'T_OUT_PARAM', 'value should be #1-#9', "Illegal: " . $v->stringify); } }
  return bless ["#$int", CC_OUT_PARAM], 'LaTeXML::Core::Token'; }
```

It is related to what you dubbed `T_XMArg`, and I think we can sync them and call them `T_Arg` as something brief and simple.

Reply:
Ah, now I recall you talking about something like `T_PARAM(1)`. It's kinda what I wanted to do! Are these the things that get substituted for in macros? (would be much cleaner)  Presumably the `####` collapsing is supposed to happen somewhere else?  The only catch is that my version explicitly & intentionally accepts non-digit!